tests/testsuite.src/syn_file.at

```diff
@@ -2282,3 +2282,74 @@ AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
 prog.cob:12: error: syntax error, unexpected INPUT, expecting Identifier
 ])
 AT_CLEANUP
+
+# Verify that FD GLOBAL will create default handlers in sub-programs, but
+# only in its own sub-programs
+
+AT_SETUP([Scope of FD GLOBAL in nested programs])
+AT_KEYWORDS([FD GLOBAL nested])
```

Comment:
all of those are already in the title (automatically in the keywords) - so I suggest to use `file` instead

Reply:
ok, done